tests/appsec/test_blocking_addresses.py

```diff
@@ -88,6 +96,7 @@ def setup_request_body_multipart(self):
     @missing_feature(context.library == "dotnet", reason="Don't support multipart yet")
     @missing_feature(context.library == "php", reason="Don't support multipart yet")
     @missing_feature(context.library == "java", reason="Happens on a subsequent WAF run")
+    @bug(context.library == "python" and context.weblog_variant == "django-poc", reason="npe")
```

Comment:
"npe" ? what does it means ?

Reply:
Ups, if I'm honest, it was a copy-paste from other `@bug` decorator :sweat_smile: 

Fixed, thanks @cbeauchesne